exo-go/src/app_dependency_helpers/index.go

```diff
@@ -0,0 +1,25 @@
+package appDependencyHelpers
+
+import (
+	"github.com/Originate/exosphere/exo-go/src/types"
+)
+
+// AppDependency contains methods that return config information about a dependency
+type AppDependency interface {
+	GetContainerName() string
+	GetDockerConfig() (types.DockerConfig, error)
```

Comment:
@charlierudolph  This function is not unit-tested yet but I will be writing [tests](https://github.com/Originate/exosphere/blob/master/exo-setup/src/app-setup.spec.ls) for `docker-compose` which will indirectly test this function. I can write unit tests here too if you want.